README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Rust](https://github.com/hello-rust/lychee/workflows/Rust/badge.svg)
 
-A fast, async, resource-friendly link checker written in Rust.  
```

Comment:
Actually added those two spaces to enforce an extra line break. 😉 
My intention was to make that sentence stand out a bit.

Reply:
Ok. A backslash (`\`) will provide the same effect. I shall update.

Reply:
And done. The line breaks should be preserved now, without trailing whitespaces!

Reply:
Ah, didn't know that `\` also works. That's nice because it makes the intent more explicit. Thanks.